Gemfile.lock

```diff
@@ -1,78 +1,78 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.0)
-      activesupport (= 6.0.0)
+    actionview (6.0.2.1)
+      activesupport (= 6.0.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activesupport (6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    builder (3.2.3)
+      zeitwerk (~> 2.2)
+    builder (3.2.4)
     concurrent-ruby (1.1.5)
-    crass (1.0.4)
+    crass (1.0.5)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.8.0)
-    i18n (1.6.0)
+    erubi (1.9.0)
+    i18n (1.7.0)
       concurrent-ruby (~> 1.0)
-    json (2.2.0)
+    json (2.3.0)
     jwt (2.1.0)
     jwt_signed_request (2.5.1)
       jwt (>= 1.5.0, < 2.2.0)
       rack
-    loofah (2.2.3)
+    loofah (2.4.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (0.9.2)
     mini_portile2 (2.4.0)
-    minitest (5.11.3)
-    money (6.13.4)
+    minitest (5.13.0)
+    money (6.13.6)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.4)
+    nokogiri (1.10.7)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.7)
+    rack (2.0.8)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.2.0)
-      loofah (~> 2.2, >= 2.2.2)
-    railties (6.0.0)
-      actionpack (= 6.0.0)
-      activesupport (= 6.0.0)
+    rails-html-sanitizer (1.3.0)
+      loofah (~> 2.3)
+    railties (6.0.2.1)
```

Comment:
### [railties](https://rubyonrails.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.0.0 → 6.0.2.1

[_[change-log](https://github.com/rails/rails/blob/v6.0.2.1/railties/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.0.2.1/railties)_]





<details>
<summary>Commits</summary>

A change of **173** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.0.0...v6.0.2.1).

These are the first 10 commits:
- (f17d964) [Support Rails with sass-rails 6](https://github.com/rails/rails/commit/f17d96475c349e3af7694015f3cc845c4fd7b297)
- (4c8c8cd) [Upgrade the maintainance policy](https://github.com/rails/rails/commit/4c8c8cd8b4c692a3186a78e1e587f416820d1b1d)
- (ebbef80) [Fix missing backtick in maintenance policy version [ci skip]](https://github.com/rails/rails/commit/ebbef80c59a459e1c9ae6f502fbf7950829a8710)
- (26aabdc) [Merge pull request #36969 from AhmedKamal20/patch-1](https://github.com/rails/rails/commit/26aabdcce88e952c9189766bae3d32d5c6448963)
- (e7bd651) [Merge pull request #36981 from tsuka/fix-tag-builder](https://github.com/rails/rails/commit/e7bd651789e01b0f78e83a6e2896821cbb146459)
- (ab91651) [Mark Rails 6.0 released notes as done [ci skip] (#36968)](https://github.com/rails/rails/commit/ab91651a54a843570a219f79c90cdd1a62231d8c)
- (55d8460) [Keep the name of the releases note consistent](https://github.com/rails/rails/commit/55d84602f9704c45bd52a5258628be9542fe8343)
- (70e6ff2) [Generate keep file in tmp/pids](https://github.com/rails/rails/commit/70e6ff2ed8596eb9bd6dcce87cb3f14015638d70)
- (3fda3f9) [Merge pull request #36985 from anmolarora/fix-take-memoizati…](https://github.com/rails/rails/commit/3fda3f9d33875e30dceb58841ee02a36a82fcaed)
- (e8257e3) [Merge pull request #36998 from jhawthorn/reaper_fork](https://github.com/rails/rails/commit/e8257e396c126c7d4012096b28fe3f280271b85f)


</details>